resources/examples/host/simpleResolver/README.md

```diff
@@ -24,14 +24,13 @@ optional arguments:
 ## Example
 
 > Note: This script requires the `openassetio` module to be available to
-> Python. Presently, this requires manual installation, see
-> [BUILDING.md](../../../../BUILDING.md) in the main OpenAssetIO
-> repository for more details.
+> Python. This can be installed via
+> `python -m pip install -r requirements.txt`.
```

Comment:
[question] It seems odd that the instructions for `simpleResolver` would have you install `openassetio` from PyPI, at least whilst it's part of this repo?

Reply:
A lot easier for people than having to build it from source?

Reply:
.. as in, if they're just looking for an example to guide their python dev

Reply:
OK that's fine, it just looked a bit tangential to this PR, and I'd imagined `simpleResolver` being used to test a build of the project, rather than treated as a standalone project, at least whilst it's in the same repo.